services/checks/checks.go

```diff
@@ -847,6 +874,22 @@ func (s *Service) groupChecksByDB(checks []check.Check) (mySQLChecks, postgreSQL
 	return
 }
 
+func filterChecksByInterval(checks []check.Check, interval check.Interval) []check.Check {
```

Comment:
// filterChecks  filters checks according to their interval buckets

Reply:
Added